src/components/generic/tooltip.js

```diff
@@ -55,11 +74,17 @@ const TooltipWrapper = styled('div')`
   }
 `
 
-export const TooltipWithText = ({ text, tooltipText, id, ...restOfProps }) => {
+export const TooltipWithText = ({
+  text,
+  tooltipText,
+  id,
+  $position = 'bottom',
```

Comment:
Should the default be 'right'?

Reply:
I made it bottom since all the other uses of the existing tooltip are bottom positioned. Didnt want to have to mess too much with them.